### ISSUE TITLE
Log spamming since v14.0

### ISSUE THREAD

**Issue by panzli** [fxsth/pledo#31]
I think since the 14.0 Release the log is quite big, as example a yesterday created file with 10 downloads has a size of 

```
root@ubuntu:/home/amnesia# wc -l /var/lib/docker/containers/2b1d61fdd2883c9e4e9e527ec54ca82591af4ec4b9a8aa3f67c5c5801c3332a2/2b1d61fdd2883c9e4e9e527ec54ca82591af4ec4b9a8aa3f67c5c5801c3332a2-json.log
27076928 /var/lib/docker/containers/2b1d61fdd2883c9e4e9e527ec54ca82591af4ec4b9a8aa3f67c5c5801c3332a2/2b1d61fdd2883c9e4e9e527ec54ca82591af4ec4b9a8aa3f67c5c5801c3332a2-json.log

root@ubuntu:/home/amnesia# ls -lisah  /var/lib/docker/containers/2b1d61fdd2883c9e4e9e527ec54ca82591af4ec4b9a8aa3f67c5c5801c3332a2/2b1d61fdd2883c9e4e9e527ec54ca82591af4ec4b9a8aa3f67c5c5801c3332a2-json.log
1320104 3,0G -rw-r----- 1 root root 3,0G Jan 21 06:43 /var/lib/docker/containers/2b1d61fdd2883c9e4e9e527ec54ca82591af4ec4b9a8aa3f67c5c5801c3332a2/2b1d61fdd2883c9e4e9e527ec54ca82591af4ec4b9a8aa3f67c5c5801c3332a2-json.log

```

Its containing the download process very detailed:

```
Download progress: 73% - 3471380640/4741284155
Download progress: 73% - 3471382080/4741284155
Download progress: 73% - 3471383520/4741284155
Download progress: 73% - 3471384960/4741284155
Download progress: 73% - 3471386400/4741284155
Download progress: 73% - 3471387840/4741284155
Download progress: 73% - 3471389280/4741284155
Download progress: 73% - 3471392160/4741284155
Download progress: 73% - 3471393600/4741284155
Download progress: 73% - 3471395040/4741284155
Download progress: 73% - 3471396480/4741284155
Download progress: 73% - 3471397920/4741284155
Download progress: 73% - 3471399360/4741284155
Download progress: 73% - 3471400800/4741284155
Download progress: 73% - 3471403680/4741284155
Download progress: 73% - 3471405120/4741284155
Download progress: 73% - 3471406560/4741284155
Download progress: 73% - 3471409440/4741284155
Download progress: 73% - 3471410880/4741284155
Download progress: 73% - 3471412320/4741284155
Download progress: 73% - 3471413760/4741284155
Download progress: 73% - 3471415200/4741284155
Download progress: 73% - 3471418080/4741284155
Download progress: 73% - 3471419520/4741284155
Download progress: 73% - 3471420960/4741284155
Download progress: 73% - 3471422400/4741284155
Download progress: 73% - 3471423840/4741284155
Download progress: 73% - 3471426720/4741284155
Download progress: 73% - 3471429600/4741284155
Download progress: 73% - 3471431040/4741284155
Download progress: 73% - 3471432480/4741284155
Download progress: 73% - 3471433920/4741284155
Download progress: 73% - 3471435360/4741284155
Download progress: 73% - 3471436800/4741284155
Download progress: 73% - 3471438240/4741284155
Download progress: 73% - 3471439680/4741284155
Download progress: 73% - 3471441120/4741284155
Download progress: 73% - 3471442560/4741284155
Download progress: 73% - 3471444000/4741284155
Download progress: 73% - 3471445440/4741284155
Download progress: 73% - 3471446880/4741284155
Download progress: 73% - 3471448320/4741284155
Download progress: 73% - 3471449760/4741284155
Download progress: 73% - 3471451200/4741284155
Download progress: 73% - 3471452640/4741284155
Download progress: 73% - 3471454080/4741284155
Download progress: 73% - 3471455520/4741284155
Download progress: 73% - 3471456960/4741284155
Download progress: 73% - 3471458400/4741284155
Download progress: 73% - 3471459840/4741284155
Download progress: 73% - 3471461280/4741284155
Download progress: 73% - 3471462720/4741284155
Download progress: 73% - 3471464160/4741284155
Download progress: 73% - 3471465600/4741284155
Download progress: 73% - 3471467040/4741284155
Download progress: 73% - 3471468480/4741284155
Download progress: 73% - 3471471360/4741284155
Download progress: 73% - 3471474240/4741284155
Download progress: 73% - 3471478336/4741284155
Download progress: 73% - 3471478560/4741284155
Download progress: 73% - 3471481440/4741284155
Download progress: 73% - 3471484320/4741284155
Download progress: 73% - 3471485760/4741284155
Download progress: 73% - 3471487200/4741284155
Download progress: 73% - 3471488640/4741284155
Download progress: 73% - 3471490080/4741284155
Download progress: 73% - 3471491520/4741284155
Download progress: 73% - 3471492960/4741284155
Download progress: 73% - 3471494400/4741284155
Download progress: 73% - 3471495840/4741284155
Download progress: 73% - 3471497280/4741284155
Download progress: 73% - 3471498720/4741284155
Download progress: 73% - 3471500160/4741284155
Download progress: 73% - 3471501600/4741284155
Download progress: 73% - 3471504480/4741284155
Download progress: 73% - 3471505920/4741284155
Download progress: 73% - 3471507360/4741284155
Download progress: 73% - 3471508800/4741284155
Download progress: 73% - 3471510240/4741284155
Download progress: 73% - 3471513120/4741284155
Download progress: 73% - 3471514560/4741284155
Download progress: 73% - 3471516000/4741284155
Download progress: 73% - 3471517440/4741284155
Download progress: 73% - 3471518880/4741284155
Download progress: 73% - 3471520320/4741284155
Download progress: 73% - 3471521760/4741284155
Download progress: 73% - 3471524640/4741284155
Download progress: 73% - 3471526080/4741284155
Download progress: 73% - 3471527520/4741284155
Download progress: 73% - 3471530400/4741284155
Download progress: 73% - 3471531840/4741284155
Download progress: 73% - 3471533280/4741284155
Download progress: 73% - 3471534720/4741284155
Download progress: 73% - 3471537600/4741284155
Download progress: 73% - 3471540480/4741284155
Download progress: 73% - 3471541920/4741284155
Download progress: 73% - 3471543360/4741284155
Download progress: 73% - 3471544800/4741284155
Download progress: 73% - 3471546240/4741284155
Download progress: 73% - 3471549120/4741284155
Download progress: 73% - 3471550560/4741284155
Download progress: 73% - 3471552000/4741284155
Download progress: 73% - 3471553440/4741284155
Download progress: 73% - 3471554880/4741284155
Download progress: 73% - 3471557760/4741284155
Download progress: 73% - 3471559200/4741284155
Download progress: 73% - 3471560640/4741284155
Download progress: 73% - 3471562080/4741284155
Download progress: 73% - 3471563520/4741284155
Download progress: 73% - 3471566400/4741284155
Download progress: 73% - 3471567840/4741284155
Download progress: 73% - 3471569280/4741284155
Download progress: 73% - 3471570720/4741284155
Download progress: 73% - 3471572160/4741284155
Download progress: 73% - 3471573600/4741284155
Download progress: 73% - 3471575040/4741284155
Download progress: 73% - 3471577920/4741284155
Download progress: 73% - 3471579360/4741284155
Download progress: 73% - 3471580800/4741284155
Download progress: 73% - 3471582240/4741284155
Download progress: 73% - 3471585120/4741284155
Download progress: 73% - 3471586560/4741284155
Download progress: 73% - 3471588000/4741284155
Download progress: 73% - 3471589440/4741284155
Download progress: 73% - 3471590880/4741284155
Download progress: 73% - 3471592320/4741284155
Download progress: 73% - 3471593760/4741284155
Download progress: 73% - 3471595200/4741284155
Download progress: 73% - 3471596640/4741284155
Download progress: 73% - 3471598080/4741284155
Download progress: 73% - 3471599520/4741284155
Download progress: 73% - 3471600960/4741284155
Download progress: 73% - 3471602400/4741284155
Download progress: 73% - 3471603840/4741284155
Download progress: 73% - 3471605280/4741284155
Download progress: 73% - 3471606720/4741284155
Download progress: 73% - 3471608160/4741284155
Download progress: 73% - 3471611040/4741284155
Download progress: 73% - 3471612480/4741284155
Download progress: 73% - 3471613920/4741284155
Download progress: 73% - 3471615360/4741284155
Download progress: 73% - 3471616800/4741284155
Download progress: 73% - 3471618240/4741284155
Download progress: 73% - 3471619680/4741284155
Download progress: 73% - 3471621120/4741284155
Download progress: 73% - 3471622560/4741284155
Download progress: 73% - 3471625440/4741284155
Download progress: 73% - 3471628320/4741284155
Download progress: 73% - 3471631200/4741284155
Download progress: 73% - 3471632640/4741284155
Download progress: 73% - 3471634080/4741284155
Download progress: 73% - 3471635520/4741284155
Download progress: 73% - 3471636960/4741284155
Download progress: 73% - 3471638400/4741284155
Download progress: 73% - 3471639840/4741284155
Download progress: 73% - 3471641280/4741284155
Download progress: 73% - 3471642720/4741284155
Download progress: 73% - 3471644160/4741284155
Download progress: 73% - 3471645600/4741284155
Download progress: 73% - 3471647040/4741284155
Download progress: 73% - 3471648480/4741284155
Download progress: 73% - 3471651360/4741284155
Download progress: 73% - 3471652800/4741284155
Download progress: 73% - 3471654240/4741284155
Download progress: 73% - 3471655680/4741284155
Download progress: 73% - 3471657120/4741284155
Download progress: 73% - 3471660000/4741284155
Download progress: 73% - 3471661440/4741284155
Download progress: 73% - 3471662880/4741284155
Download progress: 73% - 3471664320/4741284155
Download progress: 73% - 3471665760/4741284155
Download progress: 73% - 3471667200/4741284155
Download progress: 73% - 3471668640/4741284155
Download progress: 73% - 3471670080/4741284155
Download progress: 73% - 3471671520/4741284155
Download progress: 73% - 3471672960/4741284155
Download progress: 73% - 3471674400/4741284155
Download progress: 73% - 3471675840/4741284155
Download progress: 73% - 3471677280/4741284155
Download progress: 73% - 3471678720/4741284155
Download progress: 73% - 3471680160/4741284155
Download progress: 73% - 3471681600/4741284155
Download progress: 73% - 3471683040/4741284155
Download progress: 73% - 3471684480/4741284155
Download progress: 73% - 3471685920/4741284155
Download progress: 73% - 3471688800/4741284155
Download progress: 73% - 3471690240/4741284155
Download progress: 73% - 3471691680/4741284155
Download progress: 73% - 3471694560/4741284155
Download progress: 73% - 3471697440/4741284155
Download progress: 73% - 3471698880/4741284155
Download progress: 73% - 3471700320/4741284155
Download progress: 73% - 3471701760/4741284155
Download progress: 73% - 3471703200/4741284155
Download progress: 73% - 3471704640/4741284155
Download progress: 73% - 3471706080/4741284155
Download progress: 73% - 3471707520/4741284155
Download progress: 73% - 3471708960/4741284155
Download progress: 73% - 3471710400/4741284155
Download progress: 73% - 3471711840/4741284155
Download progress: 73% - 3471713280/4741284155
Download progress: 73% - 3471714720/4741284155
Download progress: 73% - 3471716160/4741284155
Download progress: 73% - 3471717600/4741284155
Download progress: 73% - 3471719040/4741284155
Download progress: 73% - 3471720480/4741284155
Download progress: 73% - 3471723360/4741284155
Download progress: 73% - 3471724800/4741284155
Download progress: 73% - 3471726240/4741284155
Download progress: 73% - 3471727680/4741284155
Download progress: 73% - 3471729120/4741284155
Download progress: 73% - 3471730560/4741284155
Download progress: 73% - 3471732000/4741284155
Download progress: 73% - 3471733440/4741284155
Download progress: 73% - 3471734880/4741284155
Download progress: 73% - 3471736320/4741284155
Download progress: 73% - 3471737760/4741284155
Download progress: 73% - 3471739200/4741284155
Download progress: 73% - 3471740640/4741284155
Download progress: 73% - 3471742080/4741284155
Download progress: 73% - 3471744960/4741284155
Download progress: 73% - 3471746400/4741284155
Download progress: 73% - 3471747840/4741284155
Download progress: 73% - 3471749280/4741284155
Download progress: 73% - 3471750720/4741284155
Download progress: 73% - 3471752160/4741284155
Download progress: 73% - 3471753600/4741284155
Download progress: 73% - 3471755040/4741284155
Download progress: 73% - 3471756480/4741284155
Download progress: 73% - 3471757920/4741284155
```

**Comment by fxsth** [comment]
Thank you @panzli for reporting!
I'll fix it right away

**Comment by fxsth** [comment]
Fixed in 1.4.1 (latest)
